thunder/torch/langctx.py

```diff
@@ -36,9 +37,13 @@ def get_method(self, id: str, *args, **kwargs) -> Callable:
                 break
         if has_tensor_input:
             method: None | Callable = _method_name_to_fn_map.get(id, None)
-            if method is None:
-                raise AttributeError(f"The {self.name} language context has no method {id}")
-            return method
+            property: None | Callable = _property_name_to_fn_map.get(id, None)
+            if method is None and property is None:
+                raise AttributeError(f"The {self.name} language context has no method or attribute {id}")
+            if method:
+                return method
+            else:
+                return property(inps[0])
```

Comment:
This looks good but don't use the name `property` because it conflicts with Python's `property` builtin:

https://docs.python.org/3/library/functions.html#property

That's why the name `property` is colored differently here

Another name might be `prop`

Reply:
Oh, I was not aware of that. Thanks! Will update it :)